heudiconv/heuristics/reproin.py

```diff
@@ -780,6 +783,13 @@ def infotoids(seqinfos, outdir):
                     "Should have got a single session marker.  Got following: %s"
                     % ', '.join(map(repr, ses_markers))
                 )
+            if re.match("(X|x)*", ses_markers):
```

Comment:
```suggestion
            if re.match("XX+$", ses_markers) or re.match("xx+$", ses_markers):
```

to
- ensure consistent `xxx` or `XXX` and not `XxX`
- require at least 2 of those xs
- no other symbol there (anchor)